perf-test/clj/reitit/perf_utils.clj

```diff
@@ -58,8 +60,10 @@
   (println)
   (suite name)
   (println)
-  (let [times (for [[path time] (bench-routes routes req f)]
+  (let [times (for [[path [mean lower]] (bench-routes routes req f)]
                 (do
-                  (when verbose? (println (format "%7s" time) "\t" path))
-                  time))]
-    (title (str "average: " (int (/ (reduce + times) (count times)))))))
+                  (when verbose? (println (format "%7s\t%7s" mean lower) "\t" path))
+                  [mean lower]))]
+    (title (str "average, lower/mean: "
+                (int (/ (reduce + (map second times)) (count times))) "/"
+                (int (/ (reduce + (map first times)) (count times)))))))
```

Comment:
What's your thinking behind looking at the lower quartiles here?

Taking the average of lower quartiles (or any percentiles or median) is not a good estimator for the lower quartiles of all the measurements. It probably works okay as long as the measured routes have similar performance, but if not, it's hard to interpret. Reitit has a pretty uniform performance, but some of the other routers have performance all over the place.

Something that could be useful here would be looking at the distributions of measurements. Since you can the measured values out of Criterium, you could print the [sparklines](https://github.com/deeplook/sparklines)-style histograms for each route and for the whole suite. A histogram can give you lot of info with a quick glance and you could even copy-paste those to the code comments!